docs/root/intro/deprecated.rst

```diff
@@ -103,6 +103,9 @@ Deprecated items below are listed in chronological order.
   `RBAC <https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/rbac/v2/rbac.proto>`_ has been deprecated
   in favor of :ref:`direct_remote_ip <envoy_api_field_config.rbac.v2.Principal.direct_remote_ip>` and
   :ref:`remote_ip <envoy_api_field_config.rbac.v2.Principal.remote_ip>`.
+* Tracing provider configuration as part of :ref:`bootstrap config <envoy_api_field_config.bootstrap.v2.Bootstrap.tracing>`
```

Comment:
Please move to 1.15.0. Also I think this changes needs a release note.

Reply:
fixed